.travis.yml

```diff
@@ -17,6 +17,13 @@ env:
     - PRODUCT_TEST_GROUP=5
     - PRODUCT_TEST_GROUP=6
     - PRODUCT_TEST_GROUP=7
+    - PRODUCT_TEST_GROUP=8
+    - PRODUCT_TEST_GROUP=9
+    - PRODUCT_TEST_GROUP=10
+    - PRODUCT_TEST_GROUP=11
+    - PRODUCT_TEST_GROUP=12
+    - PRODUCT_TEST_GROUP=13
+    - PRODUCT_TEST_GROUP=14
```

Comment:
This was to fix jobs going over 50 minutes and failing. I wasn't considering Travis memory usage as part of this PR.